buku.py

```diff
@@ -1492,6 +1492,31 @@ def get_page_title(resp):
         return htmlparser.parsed_title
 
 
+def create_poolmanager(url):
```

Comment:
> Would it work without any proxy configured as well?

My code is only configuring ProxyManager when needed. So it would still use PoolManager if there is no proxy variable defined.